ai-integrations/haystack.ipynb

```diff
@@ -116,6 +116,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from haystack_integrations.document_stores.mongodb_atlas import MongoDBAtlasDocumentStore\n",
+    "\n",
```

Comment:
I'm getting an error that the `MongoDBAtlasDocumentStore` initialization in the line below is failing due to a missing `full_text_search_index` param. Adding this param resolved this issue.